lesson15/js/main.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+const openModalBtn = document.getElementById('js-openModalBtn');
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const getModal = document.getElementById('js-modalContainer');
+		getModal.classList.add('is-visible');
+		document.body.insertAdjacentHTML(
+			'afterbegin',
+			'<div id="js-bgFixed" class="el_modalBg"></div>'
+		);
+		document.body.style.position = 'fixed';
+	},
+	false
+);
+
+function closeModal() {
+	const getModal = document.getElementById('js-modalContainer');
+	getModal.classList.remove('is-visible');
+	document.body.removeAttribute('style');
+	const modalBg = document.getElementById('js-bgFixed');
+	modalBg.remove();
+}
+
+const closeModlBtn = document.getElementById('js-closeModal');
+closeModlBtn.addEventListener(
+	'click',
+	() => {
+		closeModal();
+	},
+	false
+);
+
+function hideModalButton() {
+	const openModalBtn = document.getElementById('js-openModalBtn');
+	openModalBtn.remove();
+}
+
+const form = document.getElementById('js-loadmenuBtn');
+form.addEventListener(
+	'click',
+	(event) => {
+		event.preventDefault();
+		const inputNumber = document.getElementById('number');
+		const inputText = document.getElementById('words');
+		if (
+			inputNumber.value &&
+			inputText.value &&
+			!inputText.value.match(/\S/g) === false
```

Comment:
IMO
You can also write something like this
```suggestion
		if (
			inputNumber.value &&
			inputText.value &&
			inputText.value.match(/\S/g)
```
This is what I learned from Morita-sensei, but I think it will be easier to understand if you use [trim()](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/String/Trim).

Reply:
Thank you for review!!!!

I changed it to the trim () function that taught me to check for blanks.
Text with the blanks removed is now returned even if there are extra blanks.